pages/careers/_human-friendly.html

```diff
@@ -1,28 +1,36 @@
 <div class="row row-page-header">
   <div class="col-xs-12">
     <h2>Gruntwork is a Human-Friendly Company.</h2>
-    <p>
-      We want to build a company that is human friendly: that is, a company that is welcoming, safe, and inclusive for
-      people of all cultures, genders, and races; a company where everyone is treated fairly and with respect;
-      a company where you get to take on big challenges, but the hours are reasonable and you get plenty of time
-      off. In short: we want to work on interesting problems, with interesting people, while leading interesting lives.
-    </p>
-    <p>To be honest, we're still learning the best way to accomplish it, but here are some of the things we do today:</p>
+    <p>We aim to build a company:</p>
+    <ul>
+      <li>Where you get to take on big challenges, but the hours are reasonable.</li>
+      <li>Where everyone is treated fairly and with respect, but where disagreement is welcome.</li>
+      <li>Where the team takes great pride in our craft, but also gets things done.</li>
+      <li>Where we are all focused on advancing the company's mission, but get plenty of time off.</li>
+      <li>That is welcoming, safe, and inclusive for people of all cultures, genders, and races.</li>
+    </ul>
+    <p>In short, <strong>we want to work on interesting problems, with interesting people, while leading interesting lives.</strong></p>
+    <h3>How We Work</h3>
+    <p>We're still learning the best way to run a company, but here are some of the things we do today:</p>
     <ul>
       <li>We are transparent with just about everything within the company, including all sharing all financial details.</li>
-      <li>We determine everyone's compensation using (non-public) formulas and we share those formulas with everyone in the company so you know that everyone is being treated exactly the same way.</li>
-      <li>We have a minimum vacation policy.</li>
-      <li>Even though we are a 100% bootstrapped startup, we not only offer equity, but also pay market salaries.</li>
-      <li>Instead of having you spend a day solving artificial coding problems on a whiteboard as an interview, if we think you could be a good fit, we'll hire you for a day (or longer, if you're available) to work on a real project, using your normal coding environment, and we'll pay you for it.</li>
+      <li>We determine everyone's compensation using (non-public) formulas and we share those formulas with everyone in the company so you know that everyone is being treated exactly the same way. <a id="ref-1" href="#">[1]</a> <a id="ref-2" href="#">[2]</a></li>
+      <li>We have a minimum vacation policy. <a id="ref-3" href="#">[3]</a></li>
+      <li>We offer not only equity, but also pay above-market salaries, and offer a generous benefits package that includes 100% paid dental coverage, 100% paid vision coverage, and significant cost sharing on health coverage. <a id="ref-4" href="#">[4]</a> <a id="ref-5" href="#">[5]</a></li>
```

Comment:
As per the comment below, just replace these with real anchor tags (`<a href="citation-1">`) and they will "just work."

Reply:
Ah, I tried a few things, but didn't try that. I'll remove the code I wrote and use this simpler method.

Reply:
910bb5c76ad490434deb249764cbb9658a20c441